Tests/Calendars.Plugin.UWP.Tests/Calendars.Plugin.UWP.Tests.csproj

```diff
@@ -142,16 +142,16 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk">
-      <Version>16.4.0-preview-20191007-01</Version>
+      <Version>16.10.0</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform">
-      <Version>6.2.9</Version>
+      <Version>6.2.12</Version>
     </PackageReference>
     <PackageReference Include="MSTest.TestAdapter">
-      <Version>2.0.0</Version>
+      <Version>2.2.5</Version>
     </PackageReference>
     <PackageReference Include="MSTest.TestFramework">
-      <Version>2.0.0</Version>
+      <Version>2.2.5</Version>
```

Comment:
I'm not opposed to these changes, but I'd _generally_ prefer them to be a separate PR unless they're directly related. (and the "cause why not" suggests that they're not related)

Reply:
Sure thing. Reverted the change